.github/workflows/doc-index-updater-branch.yaml

```diff
@@ -3,7 +3,7 @@ name: doc-index-updater-branch
 on:
   push:
     branches:
-      - "*"
+      - "*/*"
```

Comment:
OK, do we need "*" as well though (for branch names that _don't_ have a slash)

Reply:
Ahh I think it should actually be `**` (https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)

Reply:
Ah yes! I was lead astray by one comment and didn't research it further :)